manifests/init.pp

```diff
@@ -7,10 +7,10 @@
 # include rclocal
 #
 class rclocal(
-  $config_file  = params_lookup( 'config_file' ),
-  $config_dir   = params_lookup( 'config_dir' ),
-  $template     = params_lookup( 'template' ),
-  $scripts      = params_lookup( 'scripts' )
+  $config_file  = $::rclocal::params::config_file,
```

Comment:
You don't need to remove params_lookup, just have a recent version installed of the puppi module